README.md

```diff
@@ -124,6 +124,52 @@ default to `0`.
 TestRPC uses a default gas limit of 4000000.  To change this set the
 environment variable `TESTRPC_GAS_LIMIT` to the desired value.
 
+#### web3.js
+
+You can implement the extended method for web3.js like this:
+```
```

Comment:
https://guides.github.com/features/mastering-markdown/

If you add the word `javascript` after these backticks then github will render this code block with syntax highlighting.